src/Time/Units.hs

```diff
@@ -36,15 +36,18 @@ module Time.Units
         -- ** Functions
        , convertUnit
        , intPart
+       , threadDelay
        ) where
 
+import Control.Monad.IO.Class (MonadIO, liftIO)
```

Comment:
You need to add `transformers` package to dependencies in order to make this work

* http://hackage.haskell.org/package/transformers

I guess `cabal` uses some global cache. That's why it didn't warn you that it doesn't know `Control.Monad.IO.Class` module. Whether we want to add dependency on `transformers` package or not is another question... But, I guess, there's not harm. `MonadIO` version is much useful as for me.